src/script/common/c_converter.cpp

```diff
@@ -346,6 +347,8 @@ aabb3f read_aabb3f(lua_State *L, int index, f32 scale)
 		lua_rawgeti(L, index, 6);
 		box.MaxEdge.Z = lua_tonumber(L, -1) * scale;
 		lua_pop(L, 1);
+	} else {
+		warningstream << "read_aabb3f: not a table, returning default 2x2x2 box." << std::endl;
```

Comment:
can't we use `log_deprecated` here?

Reply:
I am not familiar with our internal APIs yet; I appreciate the feedback!

Reply:
I have reviewed this and am removing the warning entirely because this is not part of the public Lua API, and it is used correctly everywhere in the codebase. I will document a precondition instead.